extensions/wikia/CoppaTool/CoppaToolController.class.php

```diff
@@ -140,21 +140,19 @@ public function renameIPAddress() {
 
 		$ipAddr = IP::sanitizeIP( $ipAddr );
 
-		$wikiIDs = RenameUserHelper::lookupIPActivity( $ipAddr );
+		$wikiIDs = RenameIPHelper::lookupIPActivity( $ipAddr );
 
 		$taskParams = [
 			'requestor_id' => $this->wg->User->getID(),
 			'requestor_name' => $this->wg->User->getName(),
 			'rename_user_id' => 0,
-			'rename_old_name' => $ipAddr,
-			'rename_new_name' => NON_ROUTABLE_IPV6,
-			'rename_ip' => true,
-			'notify_renamed' => false,
+			'old_ip' => $ipAddr,
+			'new_ip' => NON_ROUTABLE_IPV6,
 			'reason' => wfMessage( 'coppatool-reason' )->plain(),
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change the output format of this i18n string away from plain(). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3APlainFormatWfMessage)

[project-id]: wikia:app